app/queries/district_analytics_query.rb

```diff
@@ -35,25 +35,28 @@ def results
   end
 
   def total_registered_patients
-    @total_registered_patients ||=
-      Patient
-        .with_hypertension
-        .joins(:registration_facility)
-        .where(facilities: {id: facilities})
-        .group("facilities.id")
-        .count
-
-    return if @total_registered_patients.blank?
-
-    @total_registered_patients
-      .map { |facility_id, count| [facility_id, {total_registered_patients: count}] }
-      .to_h
+    # @total_registered_patients ||=
+    #   Patient
+    #     .with_hypertension
+    #     .joins(:registration_facility)
+    #     .where(facilities: {id: facilities})
+    #     .group("facilities.id")
+    #     .count
+
+    # return if @total_registered_patients.blank?
+
+    # @total_registered_patients
+    #   .map { |facility_id, count| [facility_id, {total_registered_patients: count}] }
+    #   .to_h
+
+    region.registered_hypertension_patients.count
```

Comment:
actually, this may need to be something like `region.registered_hypertenion_patients.group(:facility_id).count`